src/Products/PloneKeywordManager/tool.py

```diff
@@ -163,19 +163,13 @@ def getScoredMatches(self, word, possibilities, num, score, context=None):
         # Levenshtein is around, so let's use it.
         res = []
 
-        # Search for all similar terms in possibilities
-        if isinstance(word, bytes):
-            oword = str(word, encoding="utf-8")
-        else:
-            oword = word.encode("utf-8")
-
         for item in possibilities:
-            if isinstance(item, type(word)):
-                lscore = Levenshtein.ratio(word, item)
-            elif isinstance(item, type(oword)):
-                lscore = Levenshtein.ratio(oword, item)
+            if word.lower() in item.lower():
+                # if the word is a substring always include it
+                lscore = 1
```

Comment:
@gforcada this is maybe also interesting for `5.x` updates. I got a too low score (Levenshtein or difflib) when the searchterm was a substring of a keyword but too short. I now do always add it when its a substring without checking the score.

Reply:
Oh, that's nice, I build a tag comparison export one day and I did the same, show both the `Levenshtein` distance as well as _who is inside who_ list as well.

In our case, we started with +50k tags, so having too low scores was not really a problem 😓 but I see the potential.

Hopefully within next year I end up working on this 🤞🏾